plugins/source/aws/resources/services/elasticache/gen.hcl

```diff
@@ -123,4 +123,577 @@ resource "aws" "elasticache" "clusters" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSAccount"
+    }
+  }
+
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSRegion"
+    }
+  }
+}
+
+resource "aws" "elasticache" "global_replication_groups" {
+  path = "github.com/aws/aws-sdk-go-v2/service/elasticache/types.GlobalReplicationGroup"
+
+  ignoreError "IgnoreCommonErrors" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.IgnoreCommonErrors"
+  }
+
+  multiplex "AwsAccountMultiplex" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.AccountMultiplex"
+  }
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.DeleteAccountFilter"
+  }
+
+  options {
+    primary_keys = [
+      "arn"
+    ]
+  }
+
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSAccount"
+    }
+  }
+}
+
+resource "aws" "elasticache" "replication_groups" {
+  path = "github.com/aws/aws-sdk-go-v2/service/elasticache/types.ReplicationGroup"
+
+  ignoreError "IgnoreCommonErrors" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.IgnoreCommonErrors"
+  }
+
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.DeleteAccountRegionFilter"
+  }
+
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ServiceAccountRegionMultiplexer"
+    params = ["elasticache"]
+  }
+
+  options {
+    primary_keys = [
+      "arn"
+    ]
+  }
+
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSAccount"
+    }
+  }
+
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSRegion"
+    }
+  }
+
+  column "auto_minor_version_upgrade" {
+    description = "Auto minor version upgrade."
+  }
+
+  column "global_replication_group_info_global_replication_group_member_role" {
+    rename = "global_replication_group_member"
+  }
+
+  column "global_replication_group_info_global_replication_group_id" {
+    rename = "global_replication_group_id"
+  }
+
+  column "pending_modified_values_auth_token_status" {
+    skip = true
+  }
```

Comment:
It seems we are skipping all of the `pending_*` fields. Is there a reason for that?

Reply:
I kinda thought that having pending values is a bit too complicated and niche (since e.g. they will just be updated during the next fetch...)

But I can un-skip them, no issue :)

Reply:
If we are deciding between not storing some fields or not I would suggest we put the data into a JSON object. That way if a user is interested in the data they _can_ access it even if it isn't the most accessible form

Reply:
I just unskipped them for now :)